README.md

```diff
@@ -24,7 +24,7 @@
     - [Language bindings](#language-bindings)
     - [PaaS (PostgreSQL as a Service)](#paas-postgresql-as-a-service)
     - [Docker images](#docker-images)
-- [Resources](#resources)
+  - [Resources](#resources)
```

Comment:
This indentation was automatically fixed by VS Code - let me know if I should remove that change from this PR.